### PR TITLE
fix preview for /send and /request after application upgrade 

### PR DESCRIPTION
**Pull request by rasom**
another attempt to fix issue #2185 
status: ready

### CODE REVIEW

**Review by alwx — APPROVED** [review]
:+1:

**Review by oskarth — APPROVED** [review]

File: src/status_im/native_module/impl/module.cljs [hunk 1]
```diff
@@ -156,7 +156,7 @@
   "Removes duplicates by [jail path] keys, remains the last one."
   [[all-keys calls] {:keys [jail-id path] :as call}]
   (if (and (contains? all-keys [jail-id path])
-           (not= (second path) :subscription))
+           (not (#{:subscription :preview} (last path))))
```
Comment (janherich) [inline src/status_im/native_module/impl/module.cljs:159]: I actually think it's safer to do it the other way, eq only deduplicate based on `[jail-id path]` when we are sure we want to do it (eq suggestions) then the other way around.
But not a big deal for now

File: src/status_im/data_store/realm/schemas/account/v16/core.cljs [hunk 2]
```diff
@@ -36,15 +37,34 @@
...
+          (.objects "message")
+          (.filtered (str "content-type = \"" content-type "\""))
+          (.map (fn [object _ _]
+                  (let [group-id (aget object "group-id")
+                        {:keys [bot] :as content} (reader/read-string (aget object "content"))]
+                    (when-not bot
+                      (let [chat-id  (aget object "chat-id")
+                            chat     (chat-by-id realm chat-id)
+                            group?   (aget chat "group-chat")
+                            bot-name (if group?
```
Comment (janherich) [inline src/status_im/data_store/realm/schemas/account/v16/core.cljs:57]: Isn't this dangerous assumption to make ? What if we are generating for preview for command/command-request for example for command from console and not transactor-* ?

Comment (rasom) [reply src/status_im/data_store/realm/schemas/account/v16/core.cljs:57]: for the command :bot will not be empty. This is the case only for /send and /request commands atm 
It's for sure dangerous, code is ugly, etc. Though it fixes this issue...

Comment (jeluard) [reply src/status_im/data_store/realm/schemas/account/v16/core.cljs:57]: Maybe add some comments then? Let's try to reduce debt for future generations ;)

Comment (rasom) [reply src/status_im/data_store/realm/schemas/account/v16/core.cljs:57]: yes i can add comment, though migrations are one time thing, not sure that comment will be useful

**Comment by asemiankevich** [comment]
The issue is partially reproducible in latest develop upgrade (to ` 0.9.10-313-g21a80d71+ (1870)`). Case: using google play build recover account with some ETH and send 2 transactions to contact. Upgrade. Actual result: first `/send` is shown as plain text. 

image
TF https://app.testfairy.com/projects/4803622-status/builds/6954360/sessions/3/?accessToken=OD7HJXQt1-AUIkedaU5QN9GdSXM

**Pull request merged**